crypto/ml_kem/build.info

```diff
@@ -0,0 +1,5 @@
+LIBS = ../../libcrypto
+
+IF[{- !$disabled{'ml-kem'} -}]
```

Comment:
This if is also not needed I think because the crypto folder is auto excluded from the configuration if the 'no' option  is used.

Reply:
I like to retain the redundant guards in `build.info`, but removing `#ifdef` from excluded C files is probably good to do as final polish.